src/Common/src/Interop/Windows/Mincore/Interop.OpenCommPort.cs

```diff
@@ -16,5 +16,11 @@ internal partial class mincore
              uint uPortNumber,
              int dwDesiredAccess,
              int dwFlagsAndAttributes);
+
+        [DllImport(Libraries.CoreComm_L1_1_1, SetLastError = true)]
+        internal static extern SafeFileHandle OpenCommPort(
+            string portName,  // Is Sybolic Device Name Possible here? eg '\\.\USB_VID......'
```

Comment:
nit: typo:
```suggestion
            string portName,  // Is Symbolic Device Name Possible here? eg '\\.\USB_VID......'
```
Do you know what the answer is to this question? Can you please find out? I think it would be preferable having a comment stating if "it's possible" or "it's not possible", rather than an open question that may confuse future users of this method.